specs/saltpack_armor.md

```diff
@@ -255,31 +255,28 @@ ipsum](https://twitter.com/oconnor663/status/680171387353448448).
 
 ## Framing the BaseX Payload
 
-Before getting to the BaseX payload, the decoder parses the header and footer:
-
-1. Strip all whitespace and `>` characters from the message. (`>` is for
-   compatibility with email clients that use it for quoting.)
-2. Collect input up to the first period. This is the header.
-3. Assert that the header (with its whitespace stripped in step 1) matches
+The BaseX payload is surrounded ('framed') by a header and footer. Rules for parsing
+them appear here.
 
+1. Collect input up to the first period. This is the header.
+2. Assert that the header matches the regex
    ```
-   BEGIN([a-zA-Z0-9]+)?SALTPACK(ENCRYPTEDMESSAGE)|(SIGNEDMESSAGE)|(DETACHEDSIGNATURE)
+   [>\n\r\t ]+BEGIN[>\n\r\t ]+([a-zA-Z0-9]+)?[>\n\r\t ]+SALTPACK[>\n\r\t ]+(ENCRYPTED[>\n\r\t ]+MESSAGE)|(SIGNED[>\n\r\t ]+MESSAGE)|(DETACHED[>\n\r\t ]+SIGNATURE)[>\n\r\t ]+
    ```
-
-   The optional word is for an application name (like `KEYBASE`). The last two
-   words give the mode of the message.
-4. Collect input up to the second period. This is the payload. If the
-   implementation is streaming, it may decode the payload before the following
-   steps.
-5. Collect input up to the third period. This is the footer.
-6. Assert that the footer matches the header, with `END` instead of `BEGIN`.
+   The optional word is an application name (like 'KEYBASE'). The last two words give the
+   mode of the message.
+3. Collect input up to the second period. This is the payload. Before decoding, strip all
```

Comment:
Instead of saying "before decoding..." here, let's put this in the section below, to make it harder to miss.